addons/website_sale_picking/static/src/js/checkout_form.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module */
+
+import publicWidget from 'web.public.widget';
+import 'website_sale_delivery.checkout';
+
+publicWidget.registry.websiteSaleDelivery.include({
+    start: function () {
+        this.paymentOptions = document.querySelectorAll('.o_payment_option_card input[type=radio]');
+        return this._super.apply(this, ...arguments);
+    },
+
+    /**
+     * Hides or shows a payment option card.
+     * @param node the input element of the payment option card
+     * @param enabled whether to show or hide the card
+     * @private
+     */
+    _setEnablePaymentOption(node, enabled=false){
```

Comment:
`enabled` has a default value yet is never called without it being set explicitely. + linting
```suggestion
    _setEnablePaymentOption(node, enabled) {
```